src/libcollections/btree/node.rs

```diff
@@ -135,53 +135,46 @@ impl<K, V> InternalNode<K, V> {
 /// of nodes is acutally behind the box, and, partially due to this lack of information, has no
 /// destructor.
 struct BoxedNode<K, V> {
-    ptr: Unique<LeafNode<K, V>>
+    ptr: Unique<LeafNode<K, V>>,
 }
 
 impl<K, V> BoxedNode<K, V> {
     fn from_leaf(node: Box<LeafNode<K, V>>) -> Self {
-        unsafe {
-            BoxedNode { ptr: Unique::new(Box::into_raw(node)) }
-        }
+        unsafe { BoxedNode { ptr: Unique::new(Box::into_raw(node)) } }
     }
 
     fn from_internal(node: Box<InternalNode<K, V>>) -> Self {
-        unsafe {
-            BoxedNode { ptr: Unique::new(Box::into_raw(node) as *mut LeafNode<K, V>) }
-        }
+        unsafe { BoxedNode { ptr: Unique::new(Box::into_raw(node) as *mut LeafNode<K, V>) } }
     }
 
     unsafe fn from_ptr(ptr: NonZero<*const LeafNode<K, V>>) -> Self {
         BoxedNode { ptr: Unique::new(*ptr as *mut LeafNode<K, V>) }
     }
 
     fn as_ptr(&self) -> NonZero<*const LeafNode<K, V>> {
-        unsafe {
-            NonZero::new(*self.ptr as *const LeafNode<K, V>)
-        }
+        unsafe { NonZero::new(*self.ptr as *const LeafNode<K, V>) }
     }
 }
 
 /// An owned tree. Note that despite being owned, this does not have a destructor,
 /// and must be cleaned up manually.
 pub struct Root<K, V> {
     node: BoxedNode<K, V>,
-    height: usize
+    height: usize,
 }
 
-unsafe impl<K: Sync, V: Sync> Sync for Root<K, V> { }
-unsafe impl<K: Send, V: Send> Send for Root<K, V> { }
+unsafe impl<K: Sync, V: Sync> Sync for Root<K, V> {}
+unsafe impl<K: Send, V: Send> Send for Root<K, V> {}
```

Comment:
This seriously surprises me - every empty brace pair I've seen in the Rust codebase has had a space in the middle.